blockprotocol/blockprotocol
# Rename `/@user/all-types` to `/@user/types`

## 🌟 What is the purpose of this PR?

Changes profile tab slug for convenience. 

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203358502199087/1204015967491453/f) _(internal)_

## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Open `/@user/types` or manually open an user's types tab on their public profile.
2. Confirm that it's working as expected, and `@user/all-types` throws an 404 error.

## 📹 Demo

![Screenshot 2023-04-04 at 12 18 43](https://user-images.githubusercontent.com/39553853/229748382-a298a596-6ba7-4aea-a5b3-d40a9f5e67cc.png)
![Screenshot 2023-04-04 at 12 19 08](https://user-images.githubusercontent.com/39553853/229748394-2a7f87e5-3c23-421d-9323-ed7daa9c9eb4.png)
